collectors/feed/keywords.py

```diff
@@ -0,0 +1,523 @@
+# TODO tests
+
+import re
```

Comment:
FYI: I'm **not** going to refactor this file. Just add tests. If you have a different opinion, this is the place to voice it and change my plan. Thanks.

Reply:
Please file a task if that's necessary. IMO it wouldn't make sense because there was [another piece of feedback that locks the configuration to code releases](https://github.com/RedHatProductSecurity/osidb/pull/367/#discussion_r1383283922) anyway.

In any case, I think that is out of scope and would complicate and protract this story.